ui/src/views/Documentation/index.jsx

```diff
@@ -106,63 +182,220 @@ export default class Documentation extends Component {
     return this.findChildFromRootNode(rootNode) || rootNode;
   }
 
+  buildSchemaId(schemaId) {
+    if (schemaId.startsWith('/')) {
+      if (
+        process.env.TASKCLUSTER_ROOT_URL &&
+        process.env.TASKCLUSTER_ROOT_URL !== 'https://taskcluster.net'
+      ) {
+        return process.env.TASKCLUSTER_ROOT_URL + schemaId;
+      }
+
+      return `https://schemas.taskcluster.net/${schemaId.replace(
+        /^\/schemas\//,
+        ''
+      )}`;
+    }
+
+    return schemaId;
+  }
```

Comment:
This was copied from https://github.com/taskcluster/taskcluster-docs/blob/af1b485785a6ff00a7ef507d02ef7806d5fcff0d/lib/utils.js. hard-coding `taskcluster` seems odd a bit in the realm of redeployability. Thoughts on what this should be?